src/main/java/me/pesekjak/machine/logging/Console.java

```diff
@@ -107,9 +107,9 @@ public class Console implements IConsole {
     @Override
     public void log(@NotNull Level level, String... messages) {
         final String prefix = switch (level.intValue()) {
-            case 800 -> infoPrefix + ": "; // Info value
-            case 900 -> warningPrefix + ": "; // Warning value
-            case 1000 -> severePrefix + ": "; // Severe value
+            case 800 -> infoPrefix + " "; // Info value
+            case 900 -> warningPrefix + " "; // Warning value
+            case 1000 -> severePrefix + " "; // Severe value
```

Comment:
I feel like the colons added some more closure to the logging statements and without them it looks really weird

Reply:
No they do not.